src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
+        int supply = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
Consider adding a check to validate the existence of `fromFileName` before attempting to read it. This will help provide more user-friendly error messages if the file does not exist.